src/snapshot/svgtoimg.js

```diff
@@ -9,62 +9,61 @@
 
 'use strict';
 
-var EventEmitter = require('events').EventEmitter;
-
 function svgToImg(opts) {
```

Comment:
> Maybe in 2.0.0 we can move to all promises.

That's the goal. We un-exposed `Plotly.Snapshot` in `v2.0.0`.

> Are you thinking in opts something like {promise:true}

Exactly. So that `Plotly.toImage` will call `svgToImg` with `{ promise: true }` under the covers.